bin/pint

```diff
@@ -83,7 +95,7 @@ import susepubliccloudinfoclient.version as version
 
 try:
     provider_data = ifsrequest.get_provider_data(
-        None, None, 'json', 'all', None
+        None, None, result_format='json'
```

Comment:
This changes the behavior, would require a major version bump. Do we have any specific request for this change? What is the advantage to switching from `plain`, i.e. un-formatted text, to a json representation?